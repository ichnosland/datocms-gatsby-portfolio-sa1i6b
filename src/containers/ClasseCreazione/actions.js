/*
 *
 * ClasseCreazione actions
 *
 */

import {
  CLASSE_CREAZIONE_SCUOLE_ATTIVE_FETCH,
  CLASSE_CREAZIONE_SCUOLE_ATTIVE_SET,
  CLASSE_CREAZIONE_GEO_PROVINCIA_FETCH,
  CLASSE_CREAZIONE_GEO_SET,
  CLASSE_CREAZIONE_GEO_COMUNI_FETCH,
  CLASSE_CREAZIONE_GEO_COMUNI_SET,
  CLASSE_CREAZIONE_DATA_SET,
  CLASSE_CREAZIONE_DATA_POST,
  CLASSE_CREAZIONE_RESET,
  CLASSE_CREAZIONE_SPINNER_SET,
  CLASSE_CREAZIONE_FEEDBACK_SET,
  CLASSE_CREAZIONE_FEEDBACK_RESET,
  CLASSE_CREAZIONE_GEO_SCUOLE_SET,
} from './constants';


export const classeCreazioneScuoleAttiveFetch = () => ({
  type: CLASSE_CREAZIONE_SCUOLE_ATTIVE_FETCH,
});

export const classeCreazioneScuoleAttiveSet = (payload) => ({
  type: CLASSE_CREAZIONE_SCUOLE_ATTIVE_SET,
  payload,
});

export const classeCreazioneGeoProvinciaFetch = () => ({
  type: CLASSE_CREAZIONE_GEO_PROVINCIA_FETCH,
});

export const classeCreazioneGeoSet = (payload) => ({
  type: CLASSE_CREAZIONE_GEO_SET,
  payload,
});

export const classeCreazioneGeoComuneFetch = (sigla) => ({
  type: CLASSE_CREAZIONE_GEO_COMUNI_FETCH,
  sigla,
});

export const classeCreazioneGeoComuneSet = (payload) => ({
  type: CLASSE_CREAZIONE_GEO_COMUNI_SET,
  payload,
});

export const classeCreazioneGeoScuoleSet = (payload) => ({
  type: CLASSE_CREAZIONE_GEO_SCUOLE_SET,
  payload,
});

export const classeCreazioneDataSet = (payload) => ({
  type: CLASSE_CREAZIONE_DATA_SET,
  payload,
});

export const classeCreazioneDataPost = (payload, history) => ({
  type: CLASSE_CREAZIONE_DATA_POST,
  payload,
  history,
});

export const classeCreazioneReset = () => ({
  type: CLASSE_CREAZIONE_RESET,
});

export const classeCreazioneSpinnerSet = (enable) => ({
  type: CLASSE_CREAZIONE_SPINNER_SET,
  enable,
});

export const classeCreazioneFeedbackSet = (enable, tipologia, messaggio) => ({
  type: CLASSE_CREAZIONE_FEEDBACK_SET,
  enable,
  tipologia,
  messaggio,
});

export const classeCreazioneFeedbackReset = () => ({
  type: CLASSE_CREAZIONE_FEEDBACK_RESET,
});
