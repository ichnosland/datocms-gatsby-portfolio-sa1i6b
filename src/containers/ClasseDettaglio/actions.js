/*
 *
 * ClasseDettaglio actions
 *
 */

import {
  CLASSE_DETTAGLIO_ESPELLI,
  CLASSE_DETTAGLIO_RESET,
  CLASSE_DETTAGLIO_DATA_FETCH,
  CLASSE_DETTAGLIO_CORSO_FETCH,
  CLASSE_DETTAGLIO_CONTENUTO_SET,
  CLASSE_DETTAGLIO_DISPLAY_SET,
  CLASSE_DETTAGLIO_DISPLAY_SORT,
  CLASSE_DETTAGLIO_FEEDBACK_SET,
  CLASSE_DETTAGLIO_FEEDBACK_RESET,
  CLASSE_DETTAGLIO_SPINNER_SET,
} from './constants';

export const classeDettaglioEspelli = (payload) => ({
  type: CLASSE_DETTAGLIO_ESPELLI,
  payload,
});

export const classeDettaglioReset = () => ({
  type: CLASSE_DETTAGLIO_RESET,
});

export const classeDettaglioCorsoFetch = (payload) => ({
  type: CLASSE_DETTAGLIO_CORSO_FETCH,
  payload,
});

export const classeDettaglioDataFetch = (payload) => ({
  type: CLASSE_DETTAGLIO_DATA_FETCH,
  payload,
});

export const classeDettaglioContenutoSet = (payload) => ({
  type: CLASSE_DETTAGLIO_CONTENUTO_SET,
  payload,
});

export const classeDettaglioDisplaySort = (payloadData, sortingData) => ({
  type: CLASSE_DETTAGLIO_DISPLAY_SORT,
  payloadData,
  sortingData,
});

export const classeDettaglioDisplaySet = (payload) => ({
  type: CLASSE_DETTAGLIO_DISPLAY_SET,
  payload,
});

export const classeDettaglioFeedbackSet = (hasFeedback, tipologia, messaggio) => ({
  type: CLASSE_DETTAGLIO_FEEDBACK_SET,
  hasFeedback,
  tipologia,
  messaggio,
});

export const classeDettaglioFeedbackReset = () => ({
  type: CLASSE_DETTAGLIO_FEEDBACK_RESET,
});

export const classeDettaglioSpinnerSet = (enable) => ({
  type: CLASSE_DETTAGLIO_SPINNER_SET,
  enable,
});
