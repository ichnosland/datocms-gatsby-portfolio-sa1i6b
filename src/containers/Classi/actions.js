/*
 *
 * Classi actions
 *
 */

import {
  CLASSI_DATA_FETCH,
  CLASSI_DATA_SET,
  CLASSI_DATA_RESET,
  CLASSI_FEEDBACK_RESET,
  CLASSI_FEEDBACK_SET,
  CLASSI_SPINNER_SET,
} from './constants';

export const classiDataFetch = (idDisciplina) => ({
  type: CLASSI_DATA_FETCH,
  idDisciplina,
});

export const classiDataSet = (payload) => ({
  type: CLASSI_DATA_SET,
  payload,
});

export const classiDataReset = () => ({
  type: CLASSI_DATA_RESET,
});

export const classiFeedbackReset = () => ({
  type: CLASSI_FEEDBACK_RESET,
});

export const classiFeedbackSet = (enable = true, message = '', errorData = {}) => ({
  type: CLASSI_FEEDBACK_SET,
  enable,
  message,
  errorData,
});

export const classiSpinnerSet = (enable) => ({
  type: CLASSI_SPINNER_SET,
  enable,
});
