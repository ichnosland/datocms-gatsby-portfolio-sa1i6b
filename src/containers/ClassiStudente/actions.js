/*
 *
 * ClassiStudente actions
 *
 */

import {
  CLASSI_STUDENTE_FETCH,
  CLASSI_STUDENTE_SET,
  CLASSI_STUDENTE_RESET,
  CLASSI_STUDENTE_ISCRIVITI_POST,
  CLASSI_STUDENTE_SPINNER_SET,
  CLASSI_STUDENTE_FEEDBACK_SET,
  CLASSI_STUDENTE_FEEDBACK_RESET,
} from './constants';

export const classiStudenteFetch = (disciplinaId) => ({
  type: CLASSI_STUDENTE_FETCH,
  disciplinaId,
});

export const classiStudenteSet = (payload) => ({
  type: CLASSI_STUDENTE_SET,
  payload,
});

export const classiStudenteReset = () => ({
  type: CLASSI_STUDENTE_RESET,
});

export const classiStudenteIscrivitiPost = (codiceCorso, configuration) => ({
  type: CLASSI_STUDENTE_ISCRIVITI_POST,
  codiceCorso,
  configuration,
});

export const classiStudenteSpinnerSet = (enable) => ({
  type: CLASSI_STUDENTE_SPINNER_SET,
  enable,
});

export const classiStudenteFeedbackSet = (enable, tipologia, messaggio) => ({
  type: CLASSI_STUDENTE_FEEDBACK_SET,
  enable,
  tipologia,
  messaggio,
});

export const classiStudenteFeedbackReset = () => ({
  type: CLASSI_STUDENTE_FEEDBACK_RESET,
});
