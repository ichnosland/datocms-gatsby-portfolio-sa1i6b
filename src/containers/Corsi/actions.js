/*
 *
 * Corsi actions
 *
 */

import {
  CORSI_RESET,
  CORSI_ERROR_SET,
  CORSI_ERROR_RESET,
  CORSI_SPINNER_SET,
  CORSI_LIST_INITIALIZE,
  CORSI_LIST_DOCENTE_FETCH,
  CORSI_LIST_STUDENTE_FETCH,
  CORSI_LIST_SET,
  CORSI_CORSO_DETAIL_DOCENTE_TRIGGER,
  CORSI_CORSO_DETAIL_DOCENTE_SET,
} from './constants';

export const corsiReset = () => ({
  type: CORSI_RESET,
});

export const corsiSpinnerSet = (enable) => ({
  type: CORSI_SPINNER_SET,
  enable,
});

export const corsiErrorSet = (enable = true, message = '', errorData = {}) => ({
  type: CORSI_ERROR_SET,
  enable,
  message,
  errorData,
});

export const corsiErrorReset = () => ({
  type: CORSI_ERROR_RESET,
});

export const corsiListInitialize = (configuration, isDocente, userId) => ({
  type: CORSI_LIST_INITIALIZE,
  configuration,
  isDocente,
  userId,
});

export const corsiListDocenteFetch = (idDisciplina) => ({
  type: CORSI_LIST_DOCENTE_FETCH,
  idDisciplina,
});

export const corsiListStudenteFetch = (idDisciplina) => ({
  type: CORSI_LIST_STUDENTE_FETCH,
  idDisciplina,
});

export const corsiListSet = (payload) => ({
  type: CORSI_LIST_SET,
  payload,
});

export const corsiCorsoSelectedDocenteTrigger = (idCorso, configuration, isDocente, userId) => ({
  type: CORSI_CORSO_DETAIL_DOCENTE_TRIGGER,
  idCorso,
  configuration,
  isDocente,
  userId,
});

export const corsiCorsoDetailDocenteSet = (payload) => ({
  type: CORSI_CORSO_DETAIL_DOCENTE_SET,
  payload,
});
