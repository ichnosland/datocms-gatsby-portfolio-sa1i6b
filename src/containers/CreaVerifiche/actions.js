/*
 *
 * CreaVerifiche actions
 *
 */

import {
  CREA_VERIFICHE_FETCH,
  CREA_VERIFICHE_FETCH_ERROR,
  CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH,
  CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH_ERROR,
  CREA_VERIFICHE_SET,
  CREA_VERIFICHE_SPINNER_SET,
  CREA_VERIFICHE_POST,
  CREA_VERIFICHE_POST_ERROR,
  CREA_VERIFICHE_VERIFICA_SET,
  CREA_VERIFICHE_VERIFICA_ESERCIZIO_ADD,
  CREA_VERIFICHE_VERIFICA_ESERCIZIO_REMOVE,
  CREA_VERIFICHE_VERIFICA_RIEPILOGO_VIEW,
} from './constants';

export const creaverificheVerificaPostError = (messaggio = 'Impossibile inviare i dati') => ({
  type: CREA_VERIFICHE_POST_ERROR,
  messaggio,
});

export const creaverificheVerificaPost = (payload) => ({
  type: CREA_VERIFICHE_POST,
  payload,
});

export const creaverificheVerificaEserciziFetch = (prerequisito) => ({
  type: CREA_VERIFICHE_FETCH,
  prerequisito,
});

export const creaverificheVerificaEserciziFetchError = (messaggio = 'Impossibile caricare l\'elenco degli esercizi disponibili') => ({
  type: CREA_VERIFICHE_FETCH_ERROR,
  messaggio,
});

export const creaverificheVerificaEsercizioPreviewFetch = (esercizioId) => ({
  type: CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH,
  esercizioId,
});

export const creaverificheVerificaEsercizioPreviewFetchError = (
  messaggio = 'Impossibile caricare l\'elenco degli step per questo esercizio'
) => ({
  type: CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH_ERROR,
  messaggio,
});

export const creaverificheVerificaEserciziSet = (payload) => ({
  type: CREA_VERIFICHE_SET,
  payload,
});

export const creaverificheVerificaEserciziSpinnerSet = (enabled) => ({
  type: CREA_VERIFICHE_SPINNER_SET,
  enabled,
});

export const creaverificheVerificaSet = (payload) => ({
  type: CREA_VERIFICHE_VERIFICA_SET,
  payload,
});

export const creaverificheVerificaEsercizioAdd = (step) => ({
  type: CREA_VERIFICHE_VERIFICA_ESERCIZIO_ADD,
  step,
});

export const creaverificheVerificaEsercizioRemove = (step) => ({
  type: CREA_VERIFICHE_VERIFICA_ESERCIZIO_REMOVE,
  step,
});

export const creaverificheVerificaRiepilogoView = (enabled) => ({
  type: CREA_VERIFICHE_VERIFICA_RIEPILOGO_VIEW,
  enabled,
});
