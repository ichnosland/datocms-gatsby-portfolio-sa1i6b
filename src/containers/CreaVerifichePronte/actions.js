/*
 *
 * CreaVerifichePronte actions
 *
 */

import {
  CREA_VERIFICHE_PRONTE_FETCH,
  CREA_VERIFICHE_PRONTE_FETCH_ERROR,
  CREA_VERIFICHE_PRONTE_SET,
  CREA_VERIFICHE_PRONTE_SPINNER_SET,
  CREA_VERIFICHE_PRONTE_FETCH_ESERCIZI,
} from './constants';

export const creaverificheVerifichePronteFetch = (disciplinaId) => ({
  type: CREA_VERIFICHE_PRONTE_FETCH,
  disciplinaId,
});

export const creaverificheVerifichePronteFetchError = (messaggio = 'Impossibile caricare l\'elenco delle verifiche pronte') => ({
  type: CREA_VERIFICHE_PRONTE_FETCH_ERROR,
  messaggio,
});

export const creaverificheVerifichePronteSet = (payload) => ({
  type: CREA_VERIFICHE_PRONTE_SET,
  payload,
});

export const creaverificheVerifichePronteSpinnerSet = (enabled) => ({
  type: CREA_VERIFICHE_PRONTE_SPINNER_SET,
  enabled,
});

export const creaverificheVerifichePronteEserciziFetch = (payload) => ({
  type: CREA_VERIFICHE_PRONTE_FETCH_ESERCIZI,
  payload,
});
