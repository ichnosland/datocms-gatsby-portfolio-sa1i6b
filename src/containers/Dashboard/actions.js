import {
  DASHBOARD_UNITA_FILTRA,
  DASHBOARD_LIVELLI_SET,
  DASHBOARD_LIVELLI_FETCH,
  DASHBOARD_LIVELLI_FETCH_ERROR,
  DASHBOARD_SPINNER_SET,
  DASHBOARD_LIVELLI_RESET,
  DASHBOARD_MENUEGG_SET,
  DASHBOARD_SEARCHACTIVE_SET,
  DASHBOARD_LIVELLI_ISLOADING_SET,
} from './constants';

export function livelliFetch(configuration, isDocente = false, corsoId = undefined) {
  return {
    type: DASHBOARD_LIVELLI_FETCH,
    configuration,
    isDocente,
    corsoId,
  };
}

export function livelliSet(payload) {
  return {
    type: DASHBOARD_LIVELLI_SET,
    payload,
  };
}

export function livelliReset() {
  return {
    type: DASHBOARD_LIVELLI_RESET,
  };
}

export function livelliFetchError(message = 'Errore caricamento') {
  return {
    type: DASHBOARD_LIVELLI_FETCH_ERROR,
    message,
  };
}

export function livelliIsLoadingSet(value) {
  return {
    type: DASHBOARD_LIVELLI_ISLOADING_SET,
    value,
  };
}

export function spinnerSet(payload) {
  return {
    type: DASHBOARD_SPINNER_SET,
    payload,
  };
}

export function unitaFiltra(parolaChiave) {
  return {
    type: DASHBOARD_UNITA_FILTRA,
    filtro: parolaChiave,
  };
}

export const dashboardMenuEggSet = (value) => ({
  type: DASHBOARD_MENUEGG_SET,
  value,
});

export const dashboardSearchActiveSet = (value) => ({
  type: DASHBOARD_SEARCHACTIVE_SET,
  value,
});
