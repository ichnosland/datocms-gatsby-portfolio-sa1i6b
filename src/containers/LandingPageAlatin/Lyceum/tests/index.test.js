import React from 'react';
import { shallow } from 'enzyme';
import { connect } from 'react-redux';
import { compose } from 'redux';

import configureStore from 'configureStore';
import injectReducer from 'utils/injectReducer';
import reducerCurryer from 'common/applications/reducer';
import ladingPageReducer from 'containers/LandingPage/reducer';
import { landingPageToggleMenu } from 'containers/LandingPage/actions';

import LyceumComposed, { Lyceum } from '../index';

const configuration = {
  disciplinaId: 21,
  product: 'landingalatin',
  blocco: false,
  titoloApplicazione: 'Alatin Academy',
  Lyceum: '/',
  hasClassi: false,
};

const mockProps = {
  configuration,
  landingPage: {
    isMenuOpened: true,
  },
  onlandingPageToggleMenu: () => {},
  match: {
    path: '/path',
  },
};

describe('<LyceumComposed />', () => {
  it('should render an Lyceum', () => {
    const renderedComponent = shallow(<Lyceum {...mockProps} />);
    expect(renderedComponent.type()).toEqual('div');
    expect(renderedComponent).toMatchSnapshot();
  });
});


describe('<LyceumComposed />', () => {
  const withReducerConfiguration = injectReducer({ key: 'configuration', reducer: reducerCurryer(configuration) });
  const withLandingConfiguration = injectReducer({ key: 'landingPage', reducer: ladingPageReducer });
  const ConfigurationWrapper = compose(
    withReducerConfiguration,
    withLandingConfiguration,
    connect()
  )(LyceumComposed);
  const mockStore = configureStore({}, {});
  mockStore.dispatch = jest.fn();

  it('onModalSetData deve chiamare modalSetData', () => {
    const renderedComponent = shallow(
      <ConfigurationWrapper match={mockProps.match} />, { context: { store: mockStore } }
    ).dive().dive().dive();

    const receivedProps = renderedComponent.props();
    expect(receivedProps.configuration).toEqual(
      configuration
    );

    expect(mockStore.dispatch).not.toHaveBeenCalledWith(
      landingPageToggleMenu(true)
    );
    receivedProps.onlandingPageToggleMenu(true);
    expect(mockStore.dispatch).toHaveBeenCalledWith(
      landingPageToggleMenu(true)
    );
  });
});
