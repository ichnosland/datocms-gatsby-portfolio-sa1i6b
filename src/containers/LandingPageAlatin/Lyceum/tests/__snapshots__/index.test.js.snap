// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LyceumComposed /> should render an Lyceum 1`] = `
<div>
  <StickyFooter__StickyTop>
    <ComponentWithScrollPosition />
    <Section
      bgColor="#0065ad"
      first={true}
    >
      <Container>
        <Heading__H1
          color="#fff"
          margin="20px 0 -30px"
        >
          Lyceum
        </Heading__H1>
      </Container>
    </Section>
    <Section
      bgColor="#0065ad"
      padding="0 50px 50px"
    >
      <LandingCard
        action={Object {}}
        body="<p>Alatin Lyceum nasce nel solco dell’esperienza di Alatin Academy, un punto di riferimento quotidiano per oltre 15.000 studenti italiani del primo biennio del liceo che si avvicinano al latino.</p><p>Alatin Lyceum condivide di quell’esperienza la carica innovativa; l’idea di sperimentare strade nuove, l’ambizione di offrire ai docenti e agli studenti strumenti diversi per interpretare in chiave più moderna il processo di insegnamento-apprendimento della lingua latina senza mai snaturarlo nelle sue radici formative più profonde, con lo scopo di renderlo, invece, più funzionale alle esigenze di una scuola in costante e deciso cambiamento.</p><p>Alatin Lyceum recepisce nel suo impianto metodologico diversi spunti di riflessione sulla didattica del latino e sulla pratica del testo estremamente attuali, che arrivano dalle scuole aderenti alle prime sperimentazioni nell’ambito della certificazione del Latino, sulla scorta delle certificazioni di lingue vive, dal mondo accademico, attraverso la <strong>Consulta Universitaria per gli Studi Latini</strong> (<a href=\\"http://www.cusl.eu\\" target=\\"_blank\\">CUSL</a>), e dagli ambienti del MIUR.</p>"
        color="#fff"
        flexorder={2}
        src="IMAGE_MOCK"
        title="Una nuova proposta didattica"
        titleColor="#fff"
      />
    </Section>
    <Section
      bgColor="#F2F2F2"
    >
      <Container>
        <Heading__H2
          color="#0065ad"
          margin="0"
        >
          Struttura del corso
        </Heading__H2>
        <p>
          Alatin Lyceum è articolato in 
          <strong>
            6 livelli principali
          </strong>
          , a loro volta scanditi in 
          <strong>
            61 missioni complessive
          </strong>
          :
        </p>
        <ol>
          <li>
            completamento e ripasso della morfosintassi (missioni 1-20)
          </li>
          <li>
            autori dell’età repubblicana (missioni 22-30)
          </li>
          <li>
            autori dell’età augustea (missioni 31-37)
          </li>
          <li>
            autori dell’età imperiale (missioni 38-52)
          </li>
          <li>
            testi scientifici (missioni 53-60)
          </li>
          <li>
            il latino moderno (missione 61)
          </li>
        </ol>
        <p>
          Ciascuna missione comprende, in misura variabile, 
          <strong>
            due tipologie di prove
          </strong>
           distinte, potenzialmente sequenziali ma in realtà indipendenti, e dunque assegnabili alla classe singolarmente, a seconda delle necessità del docente. La missione può essere introdotta da una breve lezione teorica di argomento grammaticale (es. “L’uso dei modi indefiniti - Il participio”), o da un profilo d’autore.
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <Heading__H4
          color="#000"
        >
          Analisi e traduzione
        </Heading__H4>
        <p>
          La prima tipologia di prova, 
          <strong>
            “Analisi e traduzione”
          </strong>
          , propone una sequenza di quesiti finalizzati all’analisi dei meccanismi morfosintattici del passo, che culmina nella traduzione come momento finale, sulla falsariga di quanto già sperimentato da Alatin Academy. In questo modo la decodifica del testo latino che Alatin on-line propone è sempre preceduta da una serie di passaggi organici, che attraverso i diversi livelli del passo, quello sintattico, quello morfologico e quello semantico permettono allo studente di giungere a una comprensione del testo impossibile con qualunque altro tipo di strumento digitale.
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <Heading__H4
          color="#000"
        >
          La Certificazione delle competenze di lingua latina (CLL)
        </Heading__H4>
        <p>
          Il modello di analisi proposto da Alatin Lyceum si basa in larga parte sulle ricerche condotte dalla 
          <strong>
            Consulta Universitaria di Studi latini
          </strong>
           (CUSL), già dal 2012, sui modelli di Certificazione delle competenze di Lingua Latina. Gli studiosi che afferiscono alla CUSL hanno definito tali modelli di esercizio, distinti per livello, 
          <strong>
            A1
          </strong>
          , 
          <strong>
            A2
          </strong>
          , 
          <strong>
            B1
          </strong>
          , 
          <strong>
            B2
          </strong>
           sulla base di quelli che sono stati somministrati agli studenti di alcune regioni pilota (Lombardia, Liguria, Emilia Romagna, Sicilia), nel corso dell’ultimo quinquennio.
        </p>
        <p>
          Dopo una riflessione attenta su questo materiale, nella convinzione dell’utilità didattica degli schemi di analisi definiti dalla Consulta e ancora prima dai licei che hanno per primi sperimentato forme di certificazione nelle regioni pilota, il team e gli autori di Alatin hanno deciso di sposare le basi metodologiche di queste prove. I docenti, pertanto, ritroveranno le linee di indirizzo di questi modelli in tutte le didattiche del secondo tipo, indicate sulla dashboard come 
          <strong>
            “Prova per competenze”
          </strong>
          , corredate del relativo livello di certificazione (A o B).
        </p>
        <Heading__H4
          color="#000"
        >
          Le prove per competenze di Alatin Lyceum
        </Heading__H4>
        <p>
          Il sillabo pubblicato recentemente dalla CUSL si ispira alla struttura del QCER, il quadro europeo della certificazione linguistica, che prevede 3 livelli (A, B, C) divisi in due sottolivelli (1 e 2). Rispetto al QCER, il sillabo CUSL non prevede gli ultimi due livelli in ordine di difficoltà, C1 e C2, per l’ovvia ragione che in questi due livelli la certificazione delle lingue vive riguarda soprattutto la competenza attiva (scritta e orale), che non si considera in genere come obiettivo didattico nello studio della lingua latina.
        </p>
        <p>
          Le prove di Lyceum riprendono la distinzione in due macrolivelli A e B, che condividono, seppure a diversa gradazione, le medesime competenze.
        </p>
        <p>
          Entrambi i livelli toccano due aree di riferimento, la 
          <strong>
            comprensione globale
          </strong>
           e la 
          <strong>
            comprensione analitica
          </strong>
          , che a sua volta può essere strutturata in 
          <strong>
            competenza lessicale
          </strong>
           o 
          <strong>
            morfosintattica
          </strong>
          , quest’ultima anche “attiva”, intesa cioè come capacità di trasformare un costrutto o un’espressione latina ricorrendo a una diversa struttura sintattica.
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <p>
          A seconda del livello indicato, la prova si rifà a 
          <strong>
            due modelli
          </strong>
           di base:
        </p>
        <Lyceum__TableContainer>
          <Tables__Table
            fixed={true}
            neutral={true}
          >
            <tbody>
              <Lyceum__Tbot>
                <Lyceum__Th>
                  <strong>
                    LIVELLO A
                  </strong>
                </Lyceum__Th>
                <Lyceum__Th>
                  <strong>
                    LIVELLO B
                  </strong>
                </Lyceum__Th>
              </Lyceum__Tbot>
              <Lyceum__Tbot>
                <Lyceum__Tlat>
                  <strong>
                    Comprensione globale
                  </strong>
                  <ul>
                    <li>
                      Indica il titolo latino più adatto a caratterizzare il tema del passo
                    </li>
                    <li>
                      Scegli la sequenza che corrisponde allo sviluppo del testo
                    </li>
                    <li>
                      Individua la tipologia testuale prevalente
                    </li>
                  </ul>
                </Lyceum__Tlat>
                <Tables__Td>
                  <strong>
                    Comprensione globale
                  </strong>
                  <ul>
                    <li>
                      Completa le seguenti frasi (completamento di parafrasi)
                    </li>
                    <li>
                      Individua il punto di snodo / nuclei concettuali, le affermazioni forti / fondamentali del testo
                    </li>
                    <li>
                      Verifica le seguenti affermazioni (in latino)
                    </li>
                    <li>
                      Domanda su figure retoriche (dove possibile)
                    </li>
                  </ul>
                </Tables__Td>
              </Lyceum__Tbot>
              <Lyceum__Tbot>
                <Lyceum__Tlat>
                  <strong>
                    Competenza morfosintattica
                  </strong>
                  <ul>
                    <li>
                      Indica il valore del costrutto/dell’espressione
                    </li>
                    <li>
                      Analizza il periodo
                    </li>
                  </ul>
                </Lyceum__Tlat>
                <Tables__Td>
                  <strong>
                    Comprensione analitica (livello lessicale)
                  </strong>
                  <ul>
                    <li>
                      Indica il significato corretto del termine
                    </li>
                  </ul>
                </Tables__Td>
              </Lyceum__Tbot>
              <Lyceum__Tbot>
                <Lyceum__Tlat>
                  <strong>
                    Competenza m­orfosintattica attiva
                  </strong>
                  <ul>
                    <li>
                      Trasforma il costrutto / l’espressione
                    </li>
                  </ul>
                </Lyceum__Tlat>
                <Tables__Td>
                  <strong>
                    Comprensione analitica (livello sintattico)
                  </strong>
                  <ul>
                    <li>
                      Analizza il periodo (o domanda su struttura sintattica)
                    </li>
                    <li>
                      Indica la funzione (domanda su connettivi)
                    </li>
                    <li>
                      Domanda su rimandi anaforici
                    </li>
                    <li>
                      Trasforma il costrutto / l’espressione (competenza morfosintattica attiva)
                    </li>
                  </ul>
                </Tables__Td>
              </Lyceum__Tbot>
              <Lyceum__Tbot>
                <Lyceum__Tlat>
                  <strong>
                    Competenza lessicale
                  </strong>
                  <ul>
                    <li>
                      Indica il significato del termine
                    </li>
                  </ul>
                </Lyceum__Tlat>
                <Tables__Td />
              </Lyceum__Tbot>
              <Lyceum__Tbot>
                <Lyceum__Tlat>
                  <strong>
                    Comprensione analitica
                  </strong>
                  <ul>
                    <li>
                      Domanda in latino
                    </li>
                  </ul>
                </Lyceum__Tlat>
                <Tables__Td />
              </Lyceum__Tbot>
            </tbody>
          </Tables__Table>
        </Lyceum__TableContainer>
        <p>
          Trattandosi di schede pensate per il lavoro a casa o in classe, e che dunque consentono l’uso del dizionario, abbiamo preferito non fare uso di glosse lessicali di appoggio, né fare riferimento tutte le volte a un lessico frequenziale specifico, come invece previsto dalle prove di certificazione, dove 
          <em>
            non è consentito
          </em>
           l’uso del dizionario.
        </p>
        <p>
          Al tempo stesso, 
          <strong>
            ogni prova per competenze su Alatin Lyceum è sempre preceduta e accompagnata da quella di analisi e traduzione
          </strong>
          , che funge proprio da strumento di appoggio a livello di comprensione globale e analitica e da traduzione di “servizio” per la prova per competenze.
        </p>
        <p>
          In questo modo, il docente può sempre scegliere, sulla base dei tempi e del carico di lavoro che reputa più opportuno, di assegnare le due prove in coppia o singolarmente, a seconda che sul testo scelga di prediligere un approccio più o meno guidato dal supporto della piattaforma.
        </p>
        <Heading__H4
          color="#000"
        >
          La valutazione
        </Heading__H4>
        <p>
          Entrambe le tipologie di prove condividono la medesima reportistica, sfruttabile su diversi piani, dal colpo d’occhio all’osservazione analitica.
        </p>
        <p>
          Di ogni esercizio, sia questo versione o prova per competenze, viene analizzato l’approccio dei ragazzi a più livelli: individuale, di classe, in rapporto con la media nazionale.
        </p>
        <p>
          I grafici consentono un primo colpo d’occhio sulla prova, presa singolarmente (grafico a torta) e nel tempo (grafico a istogrammi o a curva, questi ultimi con un utilissimo confronto della classe con la media nazionale).
        </p>
        <p>
          Sotto i grafici Alatin riporta una correzione completa dell’esercizio, domanda per domanda. Il docente, usando i filtri che trova sulla colonna di sinistra, può scegliere se visualizzare la correzione per “tutta la classe” o per singolo studente.
        </p>
        <p>
          Nel primo caso, i colori che rappresentano l’andamento delle domande possono essere tre: verde per le domande a cui la maggioranza degli studenti ha risposto correttamente, giallo nel caso in cui almeno la metà degli studenti ha risposto correttamente, rosso in caso contrario.
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <p>
          Un sistema di menu a tendina permette di aprire il dettaglio delle risposte e degli studenti che le hanno fornite. Tale sistema consente di monitorare la correzione di ogni verifica concentrandosi solo sulle domande che sono risultate più problematiche, tralasciando quelle a cui la maggioranza della classe ha risposto correttamente.
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <p>
          Selezionando il nominativo dello studente sulla colonna di sinistra, Alatin fornisce la situazione dello stesso, sia per ciò che riguarda l’andamento nel tempo, dove una terza colonna in verde, relativa allo studente, compare nel grafico a istogrammi, consentendo di osservarne i risultati delle varie prove in confronto alla classe e alla media nazionale; sia per il dettaglio delle domande (e della traduzione nel caso delle versioni), che ovviamente saranno solo due tipi: corrette (in verde) e sbagliate (in rosso).
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <p>
          In alto a destra sulla pagina, un tasto con l’icona della stampante consente di stampare ed archiviare eventualmente il report, nel caso di tratti di una prova di verifica in classe, fermo restando che ciascuno studente, al momento del ritiro del compito da parte del docente riceve già, in tempo reale, il report relativo al proprio compito.
        </p>
      </Container>
    </Section>
    <Section
      bgColor="#F2F2F2"
      padding="0 50px 50px"
      paddingmobile="0 20px 30px"
    >
      <Container
        align="center"
      >
        <ButtonLink__ActionButtonLink
          bgcolor="#fe9d3e"
          margin="0 auto"
          sectionbutton={1}
          shadow={1}
          to="/contatti"
        >
          Scrivici per saperne di più
        </ButtonLink__ActionButtonLink>
      </Container>
    </Section>
  </StickyFooter__StickyTop>
  <StickyFooter__StickyBottom>
    <Footer />
  </StickyFooter__StickyBottom>
</div>
`;
